tstyche.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://tstyche.org/schemas/config.json",
+  "checkSourceFiles": true,
+  "rejectAnyType": true,
```

Comment:
Protects agains accidental `any`s. For instance a missing import can be silence with `// @ts-expect-error` and it would become `any`. Assignability tests would be happily passing, but `rejectAnyType` does catch this. Will be unable by default in the next major.